hardware/OpenWeatherMap.cpp

```diff
@@ -145,6 +145,8 @@ void COpenWeatherMap::GetMeterDetails()
 	{
 		bool bret;
 		std::string szURL = sURL.str();
+		HTTPClient::SetConnectionTimeout(5);
+		HTTPClient::SetTimeout(5);
```

Comment:
This timeouts are for safe.
All modules may set this global timeouts (logitech for example), and if other don't set this value, will use values from earlier module.